crates/holochain_conductor_api/src/config/conductor.rs

```diff
@@ -254,8 +254,8 @@ pub struct NetworkConfig {
 impl Default for NetworkConfig {
     fn default() -> Self {
         Self {
-            bootstrap_url: url2::Url2::parse("https://devtest-bootstrap-1.holochain.org"),
-            signal_url: url2::Url2::parse("wss://devtest-sbd-1.holochain.org"),
+            bootstrap_url: url2::Url2::parse("https://dev-test-bootstrap2.holochain.org"),
+            signal_url: url2::Url2::parse("wss://dev-test-bootstrap2.holochain.org"),
```

Comment:
Heads up, the old url is still used in the module-level docs at the top of this file.

Reply:
Ahh yes, fixed!